src/data/light.json

```diff
@@ -128,299 +128,364 @@
       "text": {
         "base": {
           "value": "{colors.green.600}",
-          "type": "color"
+          "type": "color",
+          "description": "Used as the text or fill colour for interactive and static primary components such as links, text, and icons"
         },
         "hover": {
           "value": "{colors.green.700}",
-          "type": "color"
+          "type": "color",
+          "description": "Used as the hovered text or fill colour for interactive primary components such as links and text buttons"
         },
         "active": {
           "value": "{colors.green.800}",
-          "type": "color"
+          "type": "color",
+          "description": "Used as the pressed text or fill colour for interactive primary components such as links and text buttons"
         }
       },
       "background": {
         "base": {
           "value": "{colors.green.600}",
-          "type": "color"
+          "type": "color",
+          "description": "Used as the background for primary components such as plain buttons. Paired with text.onBackground for contrast"
         },
         "hover": {
           "value": "{colors.green.700}",
-          "type": "color"
+          "type": "color",
+          "description": "Used as the hovered background for primary components such as plain buttons"
         },
         "active": {
           "value": "{colors.green.800}",
-          "type": "color"
+          "type": "color",
+          "description": "Used as the hovered background for primary components such as plain buttons"
         },
         "muted": {
           "value": "{colors.green.200}",
-          "type": "color"
+          "type": "color",
+          "description": "Used as the subdued background for primary components such as pills. Paired with primary.textBase for a subtler contrast"
         },
         "mutedHover": {
           "value": "{colors.green.300}",
-          "type": "color"
+          "type": "color",
+          "description": "Used as the hovered subdued background for primary components such as pills. Paired with primary.textBase for a subtler contrast"
```

Comment:
Maybe change this to 

`Used as the hovered muted background` for naming consistency with the name